.github/workflows/dormant.yml

```
@@ -0,0 +1,41 @@
+# Mark issues without activity as dormant
+# See https://github.com/actions/stale for documentation and configuration
+
+name: Mark dormant issues
+on:
+  schedule:
+    - cron: "15 2 * * *"  # trigger every day at 2:15
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          days-before-stale: 180
+          days-before-close: -1  # never close
+          stale-issue-label: ":zzz: Dormant"
+          remove-stale-when-updated: true
+          stale-issue-message: >
+            Hey, there hasn't been any activity on this for more than 180
+            days, so we have marked it as "dormant" for now until there is some
+            new activity. You are welcome to reach out to people directly by
```

```suggestion
            new activity. You are welcome to reach out to people by
```
Also, I'm thinking that the reader may wonder: "What people?" Do we mean the maintainers? Other contributors? Any contributor willing and able to provide some help?